.github/workflows/make_leo_pr.yml

```diff
@@ -1,122 +0,0 @@
-name: Make Leonardo PR
-
-on:
-  # workflow_run:
-  #   workflows: [update-and-publish-cromwell-helm-chart]
-  #   types: [completed]
-  workflow_dispatch:
```

Comment:
TOL I wonder whether leaving this action in place, but maybe with a manual trigger and no expectation of automerging on the other side, might be a good idea? It may be nice to be able to force creation of a update PR? OTOH it might be become yet another unused thing to maintain? And we can already recreate it if we need it... so ok, I've changed my mind, let's delete this for now...

Reply:
I went through the same sequence of thoughts, for what it's worth. I agree that it's easy enough to recreate it...